src/main/java/fr/uge/chargepointconfiguration/entities/Chargepoint.java

```diff
@@ -45,9 +59,9 @@ public class Chargepoint {
   @JoinColumn(name = "id_firmware", referencedColumnName = "id_firmware", nullable = false)
   private Firmware firmware;
 
-//  public int getId() {
-//    return id;
-//  }
+  public int getId() {
```

Comment:
There should be a javadoc even when the method is self explainatory...
Sorry :(

Reply:
no

Reply:
Why ???
You absolute baboon